pkg/apis/nodeup/config.go

```diff
@@ -115,8 +115,8 @@ type BootConfig struct {
 	CloudProvider kops.CloudProviderID
 	// ConfigBase is the base VFS path for config objects.
 	ConfigBase *string `json:",omitempty"`
-	// ConfigServer holds the configuration for the configuration server.
-	ConfigServer *ConfigServerOptions `json:",omitempty"`
+	// ConfigServers holds the configuration for the configuration servers.
+	ConfigServers []*ConfigServerOptions `json:",omitempty"`
```

Comment:
Not quite sure if `ConfigServers` or adding `Servers` to `ConfigServerOptions`. `Server = Servers[0]` could be kept around for backward compatibility.

Reply:
It would be better to have `Servers` in `ConfigServerOptions`. The servers aren't going to have different CAs.